.idea/hangman_ash.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you make sure these files got removed from the list of things for this merge? I dont think you have yet.